README.md

```diff
@@ -1,3 +1,61 @@
-# DAT640-2022: Information Retrieval and Text Mining course at UiS
+# DAT640-2022: Information Retrieval and Text Mining course, University of Stavanger
 
-Materials for public release (i.e., links to slides and exercises) will be shared in this repository _after_ the semester.
+This repository contains course material (lectures, labs, and assignments) for the 2022 fall edition of the course.
+
+## Course format
+
+The course followed a standard format, with lectures, assignments and group projects. Physical lecturing led by the lecturer, included discussions, Kahoot quizzes, and hands-on exercises. The lectures were not be recorded/streamed, physical attendance was expected. Individual assignments  led by the TAs were dedicated time to work on graded assignments. Student that needed help with assignments were expected to attend in person. Group projects meetings were 15 minutes time slots dedicated for getting help/feedback on progress and ideas.
+
+The semester (consisting of 13 weeks) was divided into three periods, with two [lectures](lectures/) per week and [individual assignments](assignments/) during the first three weeks, one lecture per week, individual assignments and [group project work](projects/) during the next seven weeks, and last three weeks dedicated entirely for group projects.
+Dedicated time was provided to get help with the individual assignments (from TA) as well as to get feedback on the group project (from lecturer and TA).
```

Comment:
Two periods: (1) lectures and individual assignments during the first X weeks, with group projects running in parallel, (2) last 3 weeks dedicated entirely to the group projects.

Reply:
Done